test/07-openqa-label-known-issues.t

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash -xv
+
+source test/init
+
+plan tests 16
+
+source openqa-label-known-issues
+client_args=(api --host "$host_url")
+
+export KEEP_REPORT_FILE=1
+
+client_output=''
+mock-client-output() {
+    client_output+="client_call $@"$'\n'
+    echo "$client_output\n"
+}
+
+client_call=(mock-client-output "${client_call[@]}")
+autoinst_log=$dir/test/data/04-autoinst.txt
+
+try-client-output() {
+  client_output=''
+  try "$*"' && echo "$client_output"'
+}
+openqa-cli() {
+    local id=$(basename "$4")
+    cat "$dir/test/data/$id.json"
+}
+curl() {
+    if [[ "https://openqa.opensuse.org/tests/404/file/autoinst-log.txt" = "$7" ]] || [[ "https://openqa.opensuse.org/tests/414/file/autoinst-log.txt" = "$7" ]]; then
+        echo -n "404"
+    else
+        echo -n "200"
+    fi
+}
+comment_on_job() {
+    local id=$1 comment=$2 force_result=${3:-''}
+    echo "$comment"
+}
+
+try investigate_issue
+is "$rc" 1 'id required'
+# error in tap output is from here
+
+issues="159876
+[security] test fails in krb auto_review:\"com:/tmp/nfsdir /tmp/mntdir' failed\":force_result:softfailed
+openqa-force-result
+137420
+[qe-sap] test fails in network_peering auto_review:\"az network vnet peering create.+failed\"
+action
+73375
+Job incompletes with reason auto_review:\"(?m)api failure$\" (and no further details)
+action"
+
+# test data with reason but 404 in curl respond
+id=404
+testurl="https://openqa.opensuse.org/tests/${id}"
+try-client-output investigate_issue $id
+is "$rc" 0 'investigate_issue with missing autoinst-log and with reason in job_data'
+has "$got" "seems like an old job without autoinst-log.txt" "old job without autoinst-log.txt"
+has "$got" "Job incompletes with reason" "investigation keep running without autoinst-log"
+
+# all requisite evidence are missing
+sed -i "s/api failure//" "test/data/${id}.json"
+tmplog=/tmp/autoinst-404.txt
+echo > $tmplog
```

Comment:
This and other occurrences should use quoting (even though the path doesn't contain any white-spaces right now):
```suggestion
echo > "$tmplog"
```
(You can of course skip the quoting on variable assignments like the in the next line.)